WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -28,7 +34,9 @@ final class DashboardViewModel {
     private let stores: StoresManager
     private let featureFlagService: FeatureFlagService
     private let analytics: Analytics
+    private let userDefaults: UserDefaults
     private let justInTimeMessagesManager: JustInTimeMessagesProvider
+    private var showOnboardingCancellable: AnyCancellable?
```

Comment:
nit: the latest naming convention is `*Subscription` (ref p1641438040004800-slack-C6H8C3G23)

Reply:
Done https://github.com/woocommerce/woocommerce-ios/commit/8fee921685ba23f6b0808b6f669b30bdbd8d44ed#diff-5e4b4001de415046921cf84e4ce2b6c16646ae1bb2067aa6d9ec1fbfa2a855e5R39